CodeReview/Validation/Validator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using InterviewProject.Model.Entity;
+
+namespace InterviewProject.Validation
+{
+    public class Validator
+    {
+        public List<string> GetValidationErrorsForMemberStatus(Member member, Guid memberID)
+        {
+            List<string> validationErrors = new List<string>();
+
+            if (member == null) { validationErrors.Add($"member {memberID} could not be found"); }
+            if (member.Status == "Active - Regular") { validationErrors.Add($"member {memberID} is not in a status that can be automatically adjusted"); }
+            if (!member.ProgramEnrollments.Any()) { validationErrors.Add($"member {memberID} is not enrolled in any program"); }
+            if (!member.MemberMeetingChanges.Any()) { validationErrors.Add($"member {memberID} has never had a meeting"); }
```

Comment:
should be somehow combined in logic with program enrollents to suggest their paring